packages/gafl-webapp-service/src/handlers/analytics-handler.js

```diff
@@ -49,23 +49,6 @@ export const pageOmitted = async request => {
 }
 
 export default async (request, h) => {
-  await checkAnalyticsResponse(request)
```

Comment:
Maybe I'm missing something but what is the reason for removing `checkAnalyticsResponse()` and moving that logic into the default function? The function is getting quite long as a result.

Reply:
I moved back to what was previous and will use this for when using banner at top of page, created new function underneath for specific too cookies page as needs to do it slightly different